### DISCUSSION TITLE
Antenna Which GPIO would be Shield and center

### DISCUSSION THREAD

**Discussion by trancen** [EngineerGuy314/pico-WSPRer#37] [General]
Looking to build a unit for a fixed station, couldn't find if connecting to a short run of coax to the dipole of GPIO 6 is the center of the coax or GPIO 7. 

Based on the wiki if I understood correctly. 6 is center and 7 is shield?

**Comment by bradshawlupton** [comment]
I have an opinion about this.  But we are traveling with the coolest beacon on earth. either run it for a day, then swap 6 and 7. or build two, orient them the same, 50-100' apart and run both at the same time.  "spots dont lie".  or  "the spot heard round the world"

I dont use feedline. for a single days test, at a presumed 11 mw, i run the 17 m dipole directly from gpio 6 and gpio 7.

Here, as 11 mw. one spot made it from Boston to the South of Australia.

          Here beautifully rendered with Peter Marks  VK2PTM. WSPR Watch.



Sent with Proton Mail secure email.

**Reply by serych** [reply]
It is designed to feed dipole directly, so the signal is symetrical. It would be better to use some symetrical pair to feed the antenna, but there is problem of impedance mismatch as nobody exactly knows what is output impedance of RP2040 pins, therefore any cable will yield a large CSV.   :-( 

Jakub, OK1SE

**Comment by bradshawlupton** [comment]
I observe that two devices, with just one clear change, say one dipole to the two gpio pins, and one with a “good bpf “  will have the same, more or less, spots 24 hours later, with 20% different being significant to “me”
At 11 mw, 17m wspr I can’t receive the harmonics 3000’ away. While at 5’ they are loud.

  
On Wed, Aug 28, 2024 at 5:45 AM, Jakub Šerých ***@***.***&gt; wrote:  It is designed to feed dipole directly, so the signal is symetrical. It would be better to use some symetrical pair to feed the antenna, but there is problem of impedance mismatch as nobody exactly knows what is output impedance of RP2040 pins, therefore any cable will yield a large CSV.   :-(
Jakub, OK1SE

**Comment by EngineerGuy314** [comment]
If you must use a coax feedline with a grounded shield, only use either 6 or 7 for the signal, and connect the outer coax to the GND. You will only have half the output power though. I supposed you could use a balun transformer to couple the full power into a coax. But at that rate, for a fixed station it would be easier/cheaper to just add a little bit of RF amplification.
